.gitignore

```diff
@@ -5,3 +5,7 @@ target
 .classpath
 .java-version
 *.iml
+.DS_Store
+s3/.DS_Store
```

Comment:
gitignore is smart enough to know that if you say you don't want .DS_Store, that you don't want it anywhere, so I don't think you need the last three lines here